Projects/Home/Sources/Screens/HelpCenter/HelpCenterStartView.swift

```diff
@@ -133,13 +139,15 @@ public struct HelpCenterStartView: View {
                         Spacer()
                     }
                     .withChevronAccessory
+                    .onTap {
+                        let generator = UIImpactFeedbackGenerator(style: .light)
```

Comment:
Dont need here generator if we are going to use .onTap of hRow which already use generator